content/swagger/resources/reference_entity_media_files/routes/reference_entity_media_files_code.yaml

```diff
@@ -0,0 +1,23 @@
+get:
```

Comment:
we should avoid to expose this endpoint as it's useless.

I think it's better to return the location in the header:
Location: http:/...../codeofthefile
Reference-entities-media-file-code: ....

Reply:
Done!
Now the get endpoint download directly the file